docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -3,55 +3,52 @@
 :source-highlighter: highlightjs
 
 **{ProductName}** +
-A platform to automate the process of building, testing, and releasing applications. {ProductName} offers enterprise-grade security and customizable feature sets.   
+A platform to automate the process of building, testing, and releasing applications. {ProductName} offers enterprise-grade security and customizable feature sets.
+
+**build pipeline** +
+A Tekton PipelineRun which converts from source code to a built and tested artifact in a container registry.
 
 **build pipeline customization** +
-The ability to update and manage build pipelines for each component in an application. 
+The ability to update and manage build pipelines for each component in an application.
+
+**build-time tests** +
+A specific TaskRun which is included in a build pipeline but does not required to convert from the source control to the produced artifact. This may include any number of functional or security checks produced on the source code or the produced artifact.
```

Comment:
Is it source control or source code?

How about this? Does it change the meaning?
```suggestion
A specific TaskRun within in a build pipeline that doesn't convert source code into  product artifact can include any number of functional or security checks performed on the source code or the produced artifact.
```

Reply:
It is source code in source control. Using source code might be clearer though.